data/commands_redisgraph.json

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
RedisGraph is long dead, so this file isn't needed.

Reply:
Oh ok. So it's just the four modules?

Reply:
Yes.